hooks/pre/30-upgrade.rb

```diff
@@ -141,7 +141,9 @@ def fail_and_exit(message)
                   'Please run --upgrade first, then --upgrade-puppet.'
   end
 
-  Kafo::Helpers.log_and_say :info, 'Upgrading...'
+  Kafo::Helpers.log_and_say :info, 'Upgrading, to monitor the progress on all related services, please do:'
+  Kafo::Helpers.log_and_say :info, '  foreman-tail | tee upgrade-$(date +%Y-%m-%d-%H%M).log'
+
```

Comment:
Code looks good. The only nitpick I have is that when testing this, it runs by really quickly can we add a sleep of 2-3 seconds so the user can see the command before the upgrade goes through? 

I did a sleep of 3 and seemed decent to copy the command for my 2nd terminal.